krm-functions/nad-fn/makefile

```diff
@@ -0,0 +1,21 @@
+VERSION ?= latest
```

Comment:
Please make sure makefile is in sync with the main makefile that Vish and Rado worked on.

Reply:
Well, this should just be removed...instead PR should be rebased...

Here's somewhat of a "how to", if you're not super familiar with Git:

https://github.com/nephio-project/nephio-example-packages/pull/2#pullrequestreview-1375417278

Reply:
Done